policies/branch-policy.md

```diff
@@ -0,0 +1,97 @@
+Branch Policy
+=============
+
+The openssl repository contains the following maintained branches:
+
+### The default development branch
+
+- Any type (bug fix, feature, refactoring, ...) of pull requests is allowed.
+- The development of the next minor or major release happens there.
+  Which kind of release it is is defined by OMC at some point after the
+  branch is opened after the previous release was branched.
+  Implicitly it means that any API/ABI breaking changes are not allowed
+  on this branch unless OMC defines the next release will be a major
+  release.
```

Comment:
Presumably whether or not it is a major or minor is implicit in the name, e.g. is it openssl-3.1 or openssl-4.0. We're not planning on changing the name after creation right?

Reply:
Yes, I've simplified the text here.